src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -474,28 +510,28 @@ function hideAllNewQuestionForms() {
 function prepareQuestionForm(type) {
     switch (type) {
     case 'TEXT':
-        $('#questionTypeHeader').append(FEEDBACK_QUESTION_TYPENAME_TEXT);
+        $('#questionTypeHeader').html(FEEDBACK_QUESTION_TYPENAME_TEXT);
         
         hideAllNewQuestionForms();
         break;
     case 'MCQ':
         $('#' + FEEDBACK_QUESTION_NUMBEROFCHOICECREATED + '--1').val(2);
```

Comment:
You can change this and other similar instances to `'-' + NEW_QUESTION` as well.